### ISSUE TITLE
Perpetual rotation

### ISSUE THREAD

**Issue by AlanRitchie** [cognitedata/reveal-viewer#12]
Using v5.0.7

Click and hold on the model to start rotating.
Moving the mouse causes the model to rotate as the mouse moves as expected.
Dragging the pointer out of the browser window can cause the model to start spinning even when the mouse is no longer moving, especially if dragged onto a second monitor?
Model stops spinning if you drag back onto the window before releasing the mouse.

Model keeps spinning if you release the mouse button before returning to the window, and subsequent click and drag change the rotation, but do not stop it.

**Comment by dragly** [comment]
Which browser/OS is this on? 

We just updated cognitedata/three-combo-controls to listen to window events because rotation would make an awkward jump when moved outside the window:

https://github.com/cognitedata/three-combo-controls/commit/4eb57443448b74a215992717c59d4d2c6f39ac3f

However, it might be that this caused the issue you are now seeing.

**Comment by AlanRitchie** [comment]
Chrome, Windows 10

**Comment by anderscognite** [comment]
Thanks for reporting. Will investigate.

**Comment by anderscognite** [comment]
I can't reproduce in 5.1.1. Can you verify if the issue is still there (it should be)?

**Comment by ragnhildlarsen** [comment]
Yes, weird rotation bug is still there (maybe not the same). See video: 
https://drive.google.com/file/d/1N3OmdsYtG5RzUzANMj3pBIzJ925muLdb/view?usp=sharing

If the model is repeatedly rotated in the same direction, the rotation goes into a loop and only refreshing the page stops it.
Version 5.1.1

**Comment by anderscognite** [comment]
Ahh now I get it. I can even reproduce!

**Comment by anderscognite** [comment]
This problem is caused by a too large rotation. A fix that isn't perfect is now going to staging environment and will be released tomorrow. It should prevent this from happening again.

https://github.com/cognitedata/three-combo-controls/pull/40 for details.

Will close this issue once a new version of the viewer is available.

**Comment by AlanRitchie** [comment]
The unstoppable spinning is fixed, it now just stops rotating at +/-2pi.

**Comment by anderscognite** [comment]
Yep. Let's keep this issue open until we have a proper fix for more than ± 2pi. The current implementation is better, but not perfect.
